crabpy/gateway/adressenregister.py

```diff
@@ -536,64 +542,26 @@ class Gemeente(GatewayObject):
     The smallest administrative unit in Belgium.
     """
 
-    def __init__(self, niscode, gateway, naam=AUTO, taal=AUTO, status=AUTO, uri=AUTO):
+    def __init__(self, niscode, gateway, provincie_niscode=None, namen=None, naam=None):
```

Comment:
hier nog een  def gewet(self): toevoegen, dat is dan wsl een get provincie.get gewest